src/kit/LogViewer/LogViewer.tsx

```diff
@@ -561,46 +576,71 @@ const LogViewer: React.FC<Props> = ({
           </Column>
         </Row>
       </div>
-      <div className={css.sectionBody}>
-        <Spinner center spinning={isFetching} tip={logs.length === 0 ? 'No logs to show.' : ''}>
-          <div className={css.base} ref={baseRef}>
-            <div className={css.container}>
-              <div className={css.logs} ref={refCallback}>
-                <VariableSizeList
-                  height={pageSize.height - 250}
```

Comment:
Can we keep this height as default height for the resizable component so that no major UI difference when upgrade?